docs/ref/contrib/postgres/fields.txt

```diff
@@ -566,6 +566,27 @@ the :lookup:`jsonfield.contains` lookup instead.
 If only one key or index is used, the SQL operator ``->`` is used. If multiple
 operators are used then the ``#>`` operator is used.
 
+To look for ``null`` in a JSON, use Python ``None`` as a value::
+
+    >>> Dog.objects.filter(data__owner=None)
+    <QuerySet [<Dog: Meg>]>
+
+To look for missing keys use an ``isnull`` lookup::
+
+    >>> Dog.objects.create(name='Shep', data={'breed': 'collie'})
+    >>> Dog.objects.filter(data__owner__isnull=True)
+    <QuerySet [<Dog: Shep>]>
+
+To look for cases where the key is missing or it is ``null`` use a ``Q`` object::
+
+    >>> from django.db.models import Q
+    >>> Dog.objects.filter(Q(data__owner__isnull=True) | Q(data__owner=None))
+    <QuerySet [<Dog: Meg>, <Dog: Shep>]>
+
+.. versionchanged:: 2.1
+
+    Support for a querying JSON ``null`` was added.
```

Comment:
Maybe this needs expanding to say that previously `=None` was equivalent to `isnull`. (Same with release note) ??

Reply:
Yes, and the release note goes in "backwards incompatible changes" rather than "minor features".